### PR TITLE
Small YAML changes.

### LINKED ISSUE #90: Is there a way to quickly push new config to Jenkins?

### PR DESCRIPTION
**Pull request by spxtr**
I set the update-jenkins-jobs jobs to poll github every minute and run if there is a change. That should fix #90.

I also switched to the build-discarder property instead of logrotate, since that's been yelling at us for a while.

### CODE REVIEW

File: jenkins/job-configs/kubernetes-jenkins-pull/kubernetes-update-jenkins-jobs.yaml [hunk 1]
```diff
@@ -1,11 +1,13 @@
...
-    logrotate:
-        daysToKeep: 3
+    properties:
+        - build-discarder:
+            days-to-keep: 3
     node: master
     triggers:
-        - timed: 'H/15 * * * *'
+        - pollscm:
+            cron: 'H * * * *'
```
Comment (ixdy) [inline jenkins/job-configs/kubernetes-jenkins-pull/kubernetes-update-jenkins-jobs.yaml]: I think you want '* * * * *' to run every minute?

Comment (spxtr) [inline jenkins/job-configs/kubernetes-jenkins-pull/kubernetes-update-jenkins-jobs.yaml]: Heh, done.

**Comment by ixdy** [comment]
LGTM

**Comment by gmarek** [comment]
cc @wojtek-t

**Pull request merged**